src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/index.tsx

```diff
@@ -44,11 +75,145 @@ function TriggerButton(
     badge,
     selected = false,
     highContrastHeader,
+    hasTooltip = false,
+    tooltipText,
+    hasOpenDrawer = false,
+    isMobile = false,
+    isForPreviousActiveDrawer = false,
+    isForSplitPanel = false,
   }: TriggerButtonProps,
   ref: React.Ref<ButtonProps.Ref>
 ) {
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  const tooltipValue = tooltipText ? tooltipText : ariaLabel ? ariaLabel : '';
+  const [showTooltip, setShowTooltip] = useState<boolean>(false);
+  const [suppressTooltip, setSupressTooltip] = useState<boolean>(false);
+
+  const handleTriggerClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.stopPropagation(); // Stop the event from propagating to the badge icon
+    setShowTooltip(false);
+    setSupressTooltip(true);
+    onClick(event);
+  };
+
+  const handleBlur = (keepSupressed = false) => {
+    setSupressTooltip(keepSupressed);
+    setShowTooltip(false);
+  };
+
+  const handlePointerEnter = () => {
+    setSupressTooltip(false);
+    setShowTooltip(true);
+  };
+
+  /**
+   * Takes the drawer being closed and the data-shift-focus value from a close button on that drawer that persists
+   * on the event relatedTarget to determine not to show the tooltip
+   * @param event
+   */
+  const handleOnFocus = useCallback(
+    (event: FocusEvent) => {
+      const eventWithRelatedTarget = event as any;
+      const relatedTarget = eventWithRelatedTarget?.relatedTarget;
+      const isFromAnotherTrigger = relatedTarget?.dataset?.shiftFocus === 'awsui-layout-drawer-trigger';
+      let shouldShowTooltip = false;
+      if (isForSplitPanel) {
+        const breadcrumbsComponent = document.querySelector('nav[data-awsui-discovered-breadcrumbs="true"]');
+        if (breadcrumbsComponent && relatedTarget && breadcrumbsComponent.contains(relatedTarget)) {
+          //show tooltip when using keys to navigate to it from the breadcrumbs
```

Comment:
Why is there a need to be so specific to the breadcrumbs? If the intention is to show the tooltip on focus except right after closing the split panel, can't we do something similar to what we do with the drawer triggers with `isForPreviousActiveDrawer`?

Reply:
I can do a follow up where i add a data-shift-focus to the breadcrumbs but I'd rather not touch anther part of this tooltip project.

Reply:
My concern is that this is a bit too specific, and might lead to cases where the tooltip should show and it doesn't. See, for example: if I click on the last breadcrumb, which does not have a link, the focus is still on the body but by pressing Tab the focus will move to the split panel trigger but the tooltip doesn't show up. This is because technically (i.e, according to this logic) the focus did not come from the breadcrumbs.

Reply:
thanks for the comment. Removing the check works